public/src/node.js

```diff
@@ -0,0 +1,146 @@
+// # Analyzing Data with Node
+//
+// As mentioned in the [introduction](getting_started.html), this guide is mostly geared for client-side data analysis, but with a few augmentations, the same tools can be readily used server-side with [Node](https://nodejs.org/).
+//
+// And the data is too large, this might in fact be your only option if you want to use JavaScript for your data analysis. None of your users will wait 5 minutes on your site while a giant file is loaded up, no matter how cool the analysis.
+//
+// ## Setting up a Node Project
+//
+// To get started with Node, ensure both node and [npm](https://www.npmjs.com/), the Node package manager, are installed and available via the command line:
+//
+// ```bash
+// which node
+// # /usr/local/bin/node
+// which npm
+// # /usr/local/bin/npm
+// ```
+//
+// Your paths may be different then mine, but as long as `which` returns something, you should be good to go. 
+//
+// Create a new directory for your data analysis project. In this example, we have a directory with a sub-directory called `data` which contains our `animals.tsv` file inside.
```

Comment:
I think it's worth showing the full directory structure in a code block, like this:

```
animals
  data
    animals.tsv
```